vic/drivers/image/src/vic_force.c

```diff
@@ -59,6 +59,9 @@ vic_force(void)
     size_t                     d4count[4];
     size_t                     d4start[4];
 
+    cv_converter              *converter;
```

Comment:
cv_converter is not a double (a union I assume), but we can still add it to param_set_struct like: `    cv_converter converter[N_FORCING_TYPES];`, check the units and generate the converter once. This is my preference in terms of speed (but this is probably a tiny improvement), but also to avoid repetitious warnings, etc.